crt_portal/cts_forms/templates/forms/report_details.html

```diff
@@ -1,14 +1,27 @@
 {% extends "forms/report_base.html" %}
 {% block form_questions %}
-<p class="margin-top-0"><em>{{ page_note }}</em></p>
+<p class="margin-top-1 margin-bottom-4"><em>{{ page_note }}</em></p>
 
-{% for field in wizard.form.visible_fields %}
-  <fieldset class="usa-fieldset">
-    <h3><legend class="margin-bottom-0 usa-prose-h3">{{ field.label_tag }}</legend></h3>
```

Comment:
Removing the `<h3>` is an a11y improvement based on feedback from GSA's 508 coordinator. For semantic HTML reasons, `<h3>` tags should be used for section headings, not for individual form question labels.